script/tool/test/common/package_state_utils_test.dart

```diff
@@ -289,6 +289,88 @@ void main() {
       expect(state.needsChangelogChange, true);
     });
 
+    test(
+        'does not requires changelog or version change for '
+        'non-doc-comment-only changes', () async {
+      final RepositoryPackage package =
+          createFakePlugin('a_plugin', packagesDir);
+
+      const List<String> changedFiles = <String>[
+        'packages/a_plugin/lib/a_plugin.dart',
+      ];
+
+      final GitVersionFinder git = FakeGitVersionFinder(<String, List<String>>{
+        'packages/a_plugin/lib/a_plugin.dart': <String>[
+          '-  // Old comment.',
+          '+  // New comment.',
+          '+ ', // Allow whitespace line changes as part of comment changes.
+        ]
+      });
+
+      final PackageChangeState state = await checkPackageChangeState(package,
+          changedPaths: changedFiles,
+          relativePackagePath: 'packages/a_plugin/',
+          git: git);
```

Comment:
Seems like a good time for a trailing comma. Do we have style guidelines for this?

Reply:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line seems like it covers it. The style rules there don't always apply 100% since we use the auto-formatter, but in this case it's both better under the autoformatter and follows the style guide so it's a win all around :)

I definitely don't use trailing commas as much as I should.